bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/protocol/WebSocketBase.java

```diff
@@ -53,8 +53,11 @@ public void onWebSocketClose(int statusCode, @Nullable String reason) {
 
     @Override
     public void onWebSocketError(@Nullable Throwable error) {
-        logger.debug("{} connection error: {}", this.getClass().getSimpleName(),
-                error != null ? error.toString() : null);
+        if (logger.isTraceEnabled()) {
+            logger.trace("{} connection error: {}", this.getClass().getSimpleName(), error);
+        } else {
+            logger.debug("{} connection error: {}", this.getClass().getSimpleName());
```

Comment:
Second parameter missing. But why do you do that at all? We can just log as debug all the time, why should we change it if we log at trace level?

Reply:
You're eagle eyed, thanks!
Reason is that the exception logging is really long. In case of difficult issues with connectivity it is useful, but at other times a really annoying clobberin of the logs (even at debug level).
Let me know what your preference is, I don't mind strongly either way.

Reply:
Ok, I made a choice. If you prefer differently, let me know and I will update.